src/main/java/com/microsoft/lst_bench/telemetry/SQLTelemetryRegistry.java

```diff
@@ -99,7 +99,7 @@ public void flush() throws EventException {
                           StringUtils.quote(o.getEventId()),
```

Comment:
Should we apply the new method to all parameter values, i.e., L96-102?

Reply:
I don't think it's necessary. The error occurs because of the SQL statement that is attached to the payload as part of the failure description. Those SQL statements may contain quotation marks, for example in their WHERE clauses. The event information should be well-formed and not contain any forbidden characters to start with.